downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -686,6 +718,10 @@ $= future<int> result = start name();
     >**Info:** The migration support is an experimental feature and currently only supports MySQL databases. The commands associated with the feature might change in future releases.
 
 ### Improvements
+#### EDI tool
```

Comment:
Isn't this repeating what's described above?

Reply:
Addressed with [14889b3](https://github.com/ballerina-platform/ballerina-dev-website/pull/8938/commits/14889b3d14079f1f591ba3767a169ae10bdc2607).